WooCommerce/Classes/System/WooSplitViewController.swift

```diff
@@ -5,7 +5,16 @@ import UIKit
 ///
 final class WooSplitViewController: UISplitViewController {
 
-    init() {
+    /// Convenient type for the closure to handle collapsing a split view
+    ///
+    typealias ColumnForCollapsingHandler = (UISplitViewController) -> UISplitViewController.Column
+
+    private let columnForCollapsingHandler: ColumnForCollapsingHandler
+
+    /// Init a split view with an optional handler to decide which column to collapse the split view into.
+    /// By default, always display the primary column when collapsed.
+    init(columnForCollapsingHandler: @escaping ColumnForCollapsingHandler = { _ in .primary }) {
```

Comment:
I was thinking about this default behavior and wondering if it makes more sense to default to the secondary column if it exists so that no state is lost? I think in most of the cases, if the user was viewing the detail then they probably still wanted to stay on that after rotation. Though, I'm not sure if it's easy to know whether the secondary column is shown from user selection or the default view

Reply:
So since we're using the double-column style for the split view, there have to always be 2 child view controllers. I'm keeping an [empty view controller as the secondary column initially](https://github.com/woocommerce/woocommerce-ios/blob/493acdde655d84c8c1bd23209a69bcf6f4d02893/WooCommerce/Classes/ViewRelated/Orders/OrdersSplitViewWrapperController.swift#L61) since no order is selected yet. That is the reason why I wanted to show the primary column by default, i.e on first load user must be able to see the order list and not the empty view. I hope it makes sense 😅 

I can revert the logic though - to keep the secondary column as default and only show the primary one if there's no order details controller found in the navigation stack of the secondary column. WDYT?

Reply:
> I can revert the logic though - to keep the secondary column as default and only show the primary one if there's no order details controller found in the navigation stack of the secondary column. WDYT?

Yea this sounds good to me! 👍